rules/no-jest-import.js

```diff
@@ -12,32 +12,20 @@ module.exports = {
   },
   create(context) {
     return {
-      ImportDeclaration(node) {
-        if (node.source.value === 'jest') {
-          context.report({
-            node,
-            message,
-          });
-        }
+      'ImportDeclaration[source.value="jest"]'(node) {
+        context.report({ node, message });
       },
-      CallExpression(node) {
-        const calleeName = getNodeName(node.callee);
-        if (
-          calleeName === 'require' &&
-          node.arguments[0] &&
-          node.arguments[0].value === 'jest'
-        ) {
-          context.report({
-            loc: {
-              end: {
-                column: node.arguments[0].loc.end.column,
-                line: node.arguments[0].loc.end.line,
-              },
-              start: node.arguments[0].loc.start,
+      'CallExpression[callee.name="require"][arguments.0.value="jest"]'(node) {
+        context.report({
+          loc: {
```

Comment:
can we do `loc: node.arguments[0].loc` instead of the whole thing?

Reply:
Good call, will push up a commit.